include/aspect/particle/property/interface.h

```diff
@@ -178,6 +178,19 @@ namespace aspect
           need_update () const;
 
           /**
+           * Return, which data has to be provided to update the property.
+           * Note that particle properties can only ask for no data, the
+           * solution values, and the solution gradients and all other update
+           * flags will throw exceptions.
```

Comment:
This is a little difficult to parse. Perhaps "...and the solution gradients. All other..."? Also below.